modules/contributing/pages/product-improvement-program.adoc

```diff
@@ -0,0 +1,49 @@
+= Product Improvement Program
+
+:description: Learn about the telemetry data collected by Bonita Platform and how to disable it.
+
+== Introduction
+
+Bonita Runtime and Bonita Studio collect anonymous telemetry data that help Bonitasoft teams in understanding how the product is being used and in deciding what to focus on next.
+
+The data collected is minimal, statistical in nature and cannot be used to uniquely identify a user. Please see the next section to see what data is collected and sent. Access to collected data is strictly limited to the Bonitasoft team and not shared with 3rd parties.
+
+As a growing community, we greatly appreciate the usage data users send to us, as it is very valuable in helping us make Bonita Platform a better product for everyone! However, if you are concerned about sending usage stats, you can choose to disable telemetry as described xref:contributing:product-improvement-program.adoc#disable-telemetry[here].
+
+== What data is collected?
+
+=== Runtime
+
+The runtime sends the executed connectors/actor filters id along with their input and output names. This allows us to understand which connectors are used and how they are configured.
+
+.Runtime telemetry data sample
+[source,json]
+----
+{
```

Comment:
Please add a note to reinforce the fact that we collect the 'keys' not the 'values'. If needed use a concrete example, i.e. 
If your connector is called with 
"URL":"https://mydomain.com/service" We will collect the text "URL" and not see the actual value you used.